src/identity.cc

```diff
@@ -99,6 +111,15 @@ class ModelState : public BackendModel {
   // This function is used for testing.
   TRITONSERVER_Error* CreationDelay();
 
+#ifdef TRITON_ENABLE_METRICS
+  // Setup metrics for this backend. This function is used for testing.
```

Comment:
This is an example.The custom metric will be tracked if the TRITON_ENABLE_METRICS is set true. Remove `This function is used for testing` from here and below.

Reply:
I think it will be a really good idea to document this custom metric in the ReadMe of the identity backend.